cloud/google/node_init.sh

```diff
@@ -6,4 +6,21 @@ sudo mkdir ${DATA_DIR}
 sudo /usr/share/google/safe_format_and_mount \
   -m "mkfs.ext4 -F" ${DATA_DEV} ${DATA_DIR}
 
-
+# Install google-fluentd which pushes application log files up into the Google
+# Cloud Logs Monitor.
+curl -sSO https://storage.googleapis.com/signals-agents/logging/google-fluentd-install.sh
+sha256sum ./google-fluentd-install.sh
```

Comment:
Isn't it better to just instruct the log operator to install google-fluentd themselves and assert here that it's present on their system?